src/componentes/mounted/base/Auth.tsx

```
@@ -1,19 +1,18 @@
-    }
-    return children;
-  }
-  
\ No newline at end of file
+export const Auth = ({
+	children,
+	setStatus,
+	session,
+}: Readonly<{
+	children: React.ReactNode;
+	setStatus: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)